README.md

```diff
@@ -46,6 +46,7 @@ There are several auxiliary modules available for testing:
 
 - `testutil-sessions` allows authentication to the app without using the GitHub REST API.
 - `testutil-mentions` allows retrieving new user mentions without using the GitHub REST API.
+- `testutil-client` provides the ability to run a Pingh server for testing.
```

Comment:
In the scope of the following PR, please review the description of each module to make them even shorter.

For instance, this one is 
```
- `testutil-client` is a utility code for client-side testing.
```

What it provides _right now_, does not matter. Readers will go and have a look. You have to describe the meaning and the goal. Not the particular details.